src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -161,10 +161,8 @@ class MarathonSchedulerActor private (
       }
 
     case cmd @ CancelDeployment(plan) =>
-      deploymentManager.cancel(plan).onComplete{
-        case Success(d) => self ! cmd.answer
-        case Failure(e) => logger.error(s"Failed to cancel a deployment ${plan.id} due to: ", e)
```

Comment:
Would it make sense to keep this failure logging?

Reply:
The `DeolpymentManagerActor` already logs everything. https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/core/deployment/impl/DeploymentManagerActor.scala#L282